content/general/push/activate-subscribe.textile

```diff
@@ -44,31 +44,37 @@ Before you can activate your push device or receive push notifications, you must
 
 Installation, however, is platform-specific and as such, instructions for each platform and service are provided below:
 
-h3. Install Ably for Google Firebase Cloud Messaging on Android
+h3. Install Ably for Firebase Cloud Messaging on Android
 
-As with any Firebase-enabled app, you need to include one or more services within your app to handle interactions with Firebase. You need to have a service that extends "FirebaseMessagingService":https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/FirebaseMessagingService (and overrides "@onMessageReceived()@":https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/FirebaseMessagingService) in order to handle incoming push messages; the code that handles incoming push messages does not need to interact with Ably in any way.
+To receive messages on an Android device, you need to implement a service that extends "FirebaseMessagingService":https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/FirebaseMessagingService. To handle incoming push notifications, override "@onMessageReceived()@":https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/FirebaseMessagingService). These messages (@RemoteMessage@) do not need interact with Ably, and you need to implement your own logic to handle the push notification messages appropriately, such as showing a UI "notification":https://developer.android.com/guide/topics/ui/notifiers/notifications.
```

Comment:
Typo: "do not need interact with Ably"

also, rogue closing paren after the onMessageReceived() link

Reply:
`FirebaseMessagingService` should be in code font

Reply:
Thanks, fixed by 2dc13573350c715d08fdda44b78d49035bb24b21